string avg_activation
---
string avg_activation_response